DiskIOMeter.h

```diff
@@ -16,6 +16,7 @@ typedef struct DiskIOData_ {
    uint64_t totalBytesRead;
    uint64_t totalBytesWritten;
    uint64_t totalMsTimeSpend;
+   uint64_t numDisks;
```

Comment:
Should the data type be `unsigned int`, `size_t`, `uint64_t` or `unsigned long long`?
Should the property be named "numDisks" or "nDisks"?

Other than those questions, this patch is ready.

Reply:
I think, both `uint64_t` and `size_t` would be fine. No need to change this.